README.md

```diff
@@ -1,8 +1,8 @@
 <h1 align="center">
-        <a href="https://www.lightdash.com">⚡️ Lightdash </a>
+        <a target="_blank" href="https://www.lightdash.com"><img align="center" style="max-width:300px;" src="/static/screenshots/lightdash_logo_banner.png"> </a>
 </h1>
 
-<p align="center">The open source Looker alternative</p>
+<p align="center">Open source BI for your whole team.</p>
```

Comment:
For the readme I preferred the open-source looker alternative, it feels more "this is what it does". Wdyt?